docs/contributing/introduction-to-contributing.md

```diff
@@ -20,7 +20,12 @@ Some thoughts to help you contribute to this project
 
 ## Setup
 
-TL;DR: forks are preferred.
+1. [Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the main repository at [open-sauced/insights](https://github.com/open-sauced/insights) to your own GitHub account.
```

Comment:
This might not always be true. We also encourage new contributors to contribute to open-sauced/hot instead.